templates/dag/dag.dag.condor.sub

```diff
@@ -32,7 +32,8 @@ delegate_job_GSI_credentials_lifetime = 0
 {{lines|join("\n")}}
 
 {% if generate_email_summary %}
-notify= {{email_to}}
+notify = "{{email_to}}"
+notification = "always"
```

Comment:
Also, I don't know if condor submission file parsing is case-sensitive, but the same docs as above specify that the accepted value is "Always", not "always", so perhaps this should also be changed.

Reply:
Changed to upper-case throughout...